master

kollapsis88 opened the pull request:
Решение первой домашней задачи

kollapsis88 committed:
AssertionError:

Хм, я запускал тесты с винды, у меня разделители мешали их проверке, сделал фикс

AssertionError: expected 'James\r\nRichard\r\nJohn\r\nRobert\r\nJames\r\nMichael' to equal 'James\nRichard\nJohn\nRobert\nJames\nMichael'

jsru commented:
Пожалуйста, не меняйте «системные» файлы.
          Задача не может считаться решенной, если были изменены файлы тестов или другие «системные» файлы.
          Следующие системные файлы были изменены:
```diff
- 1-module/1-task/test/index.test.js — изменен
```

kollapsis88 committed:
AssertionError:

Хм, я запускал тесты с винды, у меня разделители мешали их проверке, сделал фикс

AssertionError: expected 'James\r\nRichard\r\nJohn\r\nRobert\r\nJames\r\nMichael' to equal 'James\nRichard\nJohn\nRobert\nJames\nMichael'

jsru commented:
Поздравляю, все задачи решены абсолютно верно
```diff
Порядок вывода сообщений в консоль
+ файл с решением должен быть в папке с задачей
+ порядок вывода совпадает
```
